app/src/main/res/values/styles.xml

```diff
@@ -9,7 +13,13 @@
     </style>
 
     <!-- Base themes -->
-    <style name="Base.LightTheme" parent="Theme.AppCompat.Light.NoActionBar" />
+    <style name="Base.V7" parent="Theme.AppCompat.DayNight.NoActionBar"/>
```

Comment:
Maybe add a comment here to explain that all of this is done in order to allow overriding the styles on newer versions of android with `values-vXX/styles.xml`

Reply:
This architecture was already present in the code ([i think it was done using this answer](https://stackoverflow.com/a/40841611)) V7 stands for API version 7. `LightTheme` inherits `Base.LightTheme,` which inherits `Base.V7.LightTheme` by default (I mean if API version < 21), but if API version is 21, `Base.LightTheme` inherits `Base.V21.LightTheme`, which inherits `Base.V7.LightTheme`; if API version is 29, for example, if API version is 29, `Base.LightTheme` inherits `Base.V29.LightTheme`, which inherits `Base.V27.LightTheme`, which inherits `Base.V21.LightTheme`, which inherits `Base.V7.LightTheme` - so we don't need to repeat all the code in values, `values-v21`, `values-v27`, `values-v29`, etc.

And I think V7 can be renamed to V19, because the min API is 19, but but this will not affect the functionality.